scripts/cmake/packaging/GetDependencies.cmake

```diff
@@ -11,6 +11,12 @@ foreach(TARGET ${INSTALL_DEPENDENCIES})
     endif()
 
     if(EXISTS ${TARGET_EXE})
+        # Run CMake again after target was built to collect dependencies
```

Comment:
So, that collects the shared lib dependencies?

Reply:
This runs CMake again with the (temporary) option `PRE_INSTALL_RUN=ON` which queries the already built executables for its dependencies.